src/SamplesApp/SamplesApp.Droid/SamplesApp.Droid.csproj

```diff
@@ -43,10 +42,13 @@
     <EnableLLVM>false</EnableLLVM>
     <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
     <MandroidI18n />
-    <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a;x86;x86_64;arm64-v8a</AndroidSupportedAbis>
```

Comment:
Should this be added to Solution template as well? (without the validation, just so that all these ABIs are supported by default)

Reply:
I think so also, @jeromelaban any thoughts?

Reply:
We can. I was hesitant to do so for AOT reasons, but it definitely can cause hard to diagnose issues without it.